scripts/deploy.ts

```diff
@@ -39,6 +39,12 @@ async function main() {
         ? JSON.parse(fs.readFileSync(argv.input).toString())
         : PRODUCTION_PARAMS;
 
+    // TODO public keys passed in here
+    // pass in mnemonic seed
+    // pass in number of users
+    // argv.numUsers;
+    // argv.mnemonicSeed;
```

Comment:
@ChihChengLiang Spaced on how exactly we want to handle deployed public keys, can sync up with you to determine implementation.